package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@silintl/ui-components",
-  "version": "10.4.0",
+  "version": "10.3.0",
```

Comment:
This looks like a mistake (decrementing the minor version number, instead of incrementing it).